Sources/NetworkProtection/Storage/NetworkProtectionServerListStore.swift

```diff
@@ -170,19 +170,19 @@ public class NetworkProtectionServerListFileSystemStore: NetworkProtectionServer
         do {
             return try JSONDecoder().decode([NetworkProtectionServer].self, from: data)
         } catch {
-            try removeServerList()
```

Comment:
There's another call like this a few lines up (L 166) that should be changed too.

<img width="985" alt="Screenshot 2023-12-19 at 10 14 58 AM" src="https://github.com/duckduckgo/BrowserServicesKit/assets/1836005/16129fb2-02b3-4fb6-ab30-a9e8f3934d0f">

Reply:
Whoops, good catch. Thanks!